bgpd/bgpd.h

```diff
@@ -678,9 +678,13 @@ struct bgp {
 
 	/* Weighted ECMP related config. */
 	enum bgp_link_bw_handling lb_handling;
-
-	/* Process Queue for handling routes */
+  
```

Comment:
please no whitespaces here

Reply:
sorry, i have resolved the conflict in GUI. not addressed your previous comments yet due to VM issues. will send it by EOD

Reply:
addressed this comment